v2/component/runtime_info.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package component
+
+import (
+	"encoding/json"
+
+	pb "github.com/kubeflow/pipelines/v2/third_party/ml_metadata"
+)
+
+type inputParameter struct {
+	ParameterType  string
+	ParameterValue string
+}
+
+type inputArtifact struct {
+	// Where to read MLMD artifact. File is passed using Argo artifacts.
+	FileInputPath string
```

Comment:
nit: shall we name it `MetadataInputPath` and `MetadataOutputPath`?

I think that makes it clearer the artifact payload file isn't going into this path.

Reply:
Yes, that makes sense. This change will require us to change compiler too. I'm leaving a TODO here for now, and I'll send a separate PR that renames both. Is that ok with you?